Embassy.xcodeproj/project.pbxproj

```diff
@@ -552,6 +552,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
 				en,
```

Comment:
Do we want both "English" and "en" here?

Reply:
@nealyoung No, `English` is deprecated language. 
Notice I haven't fixed any warnings on this PR because there's another swift 5 branch which (I guess) will be used for publishing?